docs/guide/setup/creating_ligand_networks.rst

```diff
@@ -3,4 +3,68 @@
 Defining the Ligand Network
```

Comment:
maybe add a cross ref tag (similar to others in `guide/`) so people can cross reference this page too